ignite/cmd/cmd.go

```diff
@@ -11,10 +11,8 @@ import (
 	"time"
 
 	"github.com/fatih/color"
-	"github.com/spf13/cobra"
-	flag "github.com/spf13/pflag"
-
-	"github.com/ignite-hq/cli/ignite/pkg/clispinner"
+	"github.com/ignite-hq/cli/ignite/pkg/cliui/clispinner"
```

Comment:
@ivanovpetr, is there a problem with your formatter?

Because previously these import paths grouped by as 3rd party ones and ignite cli ones.

Can you please run `make format` and see if it'll make a difference? Maybe your formatter's version is old or you're using a different one.

Reply:
I did'not use formatter at this stage. I just wanted to get feedback about package itself